components/calendar/src/ethiopic.rs

```diff
@@ -208,8 +208,8 @@ impl Date<Ethiopic> {
         Ok(Date::from_raw(EthiopicDateInner(inner, false), Ethiopic))
     }
 
-    pub fn set_amete_alem(&self, value: bool) -> Date<Ethiopic> {
-        Date::from_raw(EthiopicDateInner(self.inner().0, value), Ethiopic)
+    pub fn set_amete_alem(&mut self, value: bool) {
+        self.inner_mut.set_amete_alem(value);
```

Comment:
should just be `self.inner`, inner_mut was a suggestion i had in case we were to add an `inner_mut()` method, but we don't need to

Reply:
Thats an old commit, the latest commit fixes this